releasenotes/notes/logs-fix-container-log-rotation-0f15727384e0048a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Properly handle docker container logs in multiline mode in case of infrequence log messages, log file rotations or agent restart
```

Comment:
You could be even more specific and explain what would happen in such a case, i.e resending a chunked log